content/en/docs/setup/additional-setup/external-controlplane/index.md

```diff
@@ -191,6 +191,7 @@ EOF
 Apply the configuration on `external_cluster`:
 
 {{< text bash >}}
+$ kubectl create namespace external-istiod --context="${CTX_EXTERNAL_CLUSTER}
```

Comment:
```suggestion
$ kubectl create namespace external-istiod --context="${CTX_EXTERNAL_CLUSTER}"
```

Reply:
Actually, we don't need this command...I was missing this namespace due to a bug in a different step